recipes/llvmdev38/build.sh

```diff
@@ -0,0 +1,38 @@
+CMAKE_COMMON_VARIABLES=" -DCMAKE_INSTALL_PREFIX=$PREFIX/lib/llvm-3.8 \
```

Comment:
Instead of setting this to `$PREFIX/lib/llvm-3.8` maybe we can set it to `$PREFIX` and then move just the libraries after the fact to `$PREFIX/lib/llvm-3.8`. That should give us the desired effect without moving the headers and binaries into this location too.